packages/office-ui-fabric-react/package.json

```diff
@@ -22,6 +22,7 @@
     "@types/enzyme": "^2.7.5",
```

Comment:
Woops. Recently there was an update to rush, which is not backwards compatible and may not work.

Instead, you can use the locally installed copy by running this from the root:

```bash
npm run change
```